content/kotlin/concepts/datetime/terms/daysUntil/daysUntil.md

```diff
@@ -17,36 +17,31 @@ The **`.daysUntil()`** method calculates the number of whole days between two in
 ## Syntax
 
 ```pseudo
-fun Instant.daysUntil(other: Instant, timeZone: TimeZone): Int
+firstInstant.daysUntil(secondInstant, TimeZone)
 ```
 
-- `Instant`: A moment in time.
-- `TimeZone`: This parameter is of type `TimeZone`. It indicates that the calculation should consider the time zone when determining the number of days between the two `Instant` objects.
+- `firstInstant`: The first reference time.
+- `secondInstant`: The second time.
+- `TimeZone`: The timezone that the calculation should consider zone when determining the number of days between the two `Instant` objects.
 
 ## Example
 
 The example demonstrates the use of `.daysUntil()` to calculate the number of days between two `Instants`.
 
 ```kotlin
-import java.time.Instant
-import java.time.Duration
-fun Instant.daysUntil(other: Instant): Long {
-// Calculate the duration between 'this' Instant and 'other' Instant
-    val duration = Duration.between(this, other)
-
-// Convert the duration to days and return the result
-    return duration.toDays()
-}
+import kotlinx.datetime.*
+
 fun main() {
-    val t1 = Instant.parse("2008-09-29T12:00:00Z")
-    val t2 = t1.plus(Duration.ofHours(125))
-    val t3 = t2.daysUntil(t1)
+    val t1 = Instant.parse("2024-01-01T12:00:00Z")
+    val t2 = Instant.parse("2023-11-01T12:00:00Z")
+    val defaultTZ = TimeZone.currentSystemDefault()
+    val t3 = t2.daysUntil(t1, defaultTZ)
```

Comment:
Everything is fine but can we use better variable names? Like startTime endTime and t3 is basically diff btw t1 n t2 so variable name related to that

Reply:
Deferring merge so @caupolicandiaz can address this.

Reply:
Yep